### PR TITLE
feat: added landing and created folder structure

### PR DESCRIPTION
**Pull request by pradipgharge**
Added the landing page for component library and made the folder structure.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,115 @@
...
+                <h2>Designed by creatives, built for {coders} !</h2>
+            </div>
+            <div class="feature-para">
+                <p>Break the design barier, Linear UI is designed for speed & cutomization. Experience true flexiblity,
+                    aesthetic components and build projects without boundaries.</p>
+            </div>
+            <div class="feature-cards">
+                <div class="feature-cards-card">
+
+                    <img src="/Assets/images/Browsing.svg" alt="Brosing image">
```
Comment (Naman-Saxena1) [inline index.html]: Browsing image is incorrect in alt attribute

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Looks good to me 👍

**Review by shangkaul — APPROVED** [review]
Pradip, the landing page looks great! Some suggestions-
- You can add :active/:hover sudo class to nav items and buttons so user gets some feed back on hover/click.
- You can give a min-width to the body to make it responsive for larger screens. (Current approach of computed margins also looks good)

File: index.html (hunk 1 above)
Comment (pradipgharge) [reply index.html]: Ok I will fix it, Thanks for the review

**Comment by pradipgharge** [comment]
> Pradip, the landing page looks great! Some suggestions-
> 
> * You can add :active/:hover sudo class to nav items and buttons so user gets some feed back on hover/click.
> * You can give a min-width to the body to make it responsive for larger screens. (Current approach of computed margins also looks good)

Thanks Shivang for the suggestions, I will try to add these changes !

**Pull request merged**